app/assets/scss/overrides/_task-list.scss

```diff
@@ -1,82 +1,109 @@
+    @include govuk-font($size: 19);
+    padding-left: 0;
+    @include govuk-media-query($from: tablet) {
+      padding-left: govuk-spacing(6);
+    }
+  }
 
-    a {
-      border-bottom: 1px solid $border-colour;
-      display: block;
-      padding: $gutter-half 0;
-      padding-right: 20%;
-      position: relative;
+  &__links {
+    padding-left: 0;
+    @include govuk-media-query($from: tablet) {
+      padding-left: govuk-spacing(6);
+    }
+  }
 
-      &:hover {
-        color: $link-hover-colour;
-      }
+  &__link {
+    display: inline-block;
+    @include govuk-font($size: 19);
+  }
 
-      &:focus {
-        outline: none;
-        color: $black;
-        box-shadow: -3px 0 0 0 $focus-colour, 3px 0 0 0 $focus-colour;
-        border-color: transparent;
-        top: -1px;
-        margin-bottom: -2px;
-        padding-top: $gutter-half + 1px;
-        padding-bottom: $gutter-half + 1px;
-      }
+  &__items {
+    @include govuk-font($size: 19);
+    list-style: none;
+    padding-left: 0;
+    @include govuk-media-query($from: tablet) {
+      padding-left: govuk-spacing(6);
+    }
+  }
 
+  &__items.dm-task-list {
+    @include govuk-media-query($from: tablet) {
+      padding-left: 0;
     }
 
-    span.dm-task-list__text {
-      border-bottom: 1px solid $border-colour;
-      display: block;
-      padding: $gutter-half 0;
-      padding-right: 20%;
-      position: relative;
+    .dm-task-list__text {
+      @include govuk-media-query($from: tablet) {
+        padding-left: 0;
+      }
     }
 
+    padding-bottom: govuk-spacing(6);
   }
 
-  &-indicator-completed {
-    @extend %dm-task-list-indicator;
-    background-color: govuk-shade(govuk-colour("blue"), 30);
-    color: $grey-4;
-    // Just a pinch of letter spacing to make reversed-out text a bit
-    // easier to read
-    letter-spacing: 0.02em;
+  &__item {
+    border-bottom: 1px solid $govuk-border-colour;
+    margin-bottom: 0 !important;
+    padding-top: govuk-spacing(2);
+    padding-bottom: govuk-spacing(2);
+    @include govuk-clearfix;
   }
 
-  &-indicator-not-completed {
-    @extend %dm-task-list-indicator;
-    background-color: govuk-tint(govuk-colour("blue"), 80);
-    color: govuk-shade(govuk-colour("blue"), 30);
+  &__item:first-child {
+    border-top: 1px solid $govuk-border-colour;
   }
 
-  &-indicator-todo {
-    @extend %dm-task-list-indicator;
-    background-color: govuk-tint(govuk-colour("grey-1"), 90);
-    color: $black;
+  &__task-name {
+    display: block;
+    @include govuk-media-query($from: 450px) {
+      float: left;
+    }
   }
 
+  &__tag {
+    margin-top: govuk-spacing(2);
+    margin-bottom: govuk-spacing(1);
+  
+    @include govuk-media-query($from: 450px) {
+      float: right;
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+    /**
+     * We want to override the govuk tag behaviour of capitalising text.
+     * This rule should be kept when we upgrade to v3.
+     */
+    text-transform: none;
```

Comment:
Why do we want to override this behaviour?

Reply:
We don't! We did this on the Briefs Frontend, and I need to rip this out now that we're conforming more with the Design System.